NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2020 Composewell Technologies
```

Comment:
Write the NOTICE file in the format suggested [here](https://www.apache.org/legal/src-headers.html#notice) and [here](https://infra.apache.org/licensing-howto.html#mod-notice).

The following should be the full content of this file as of now.

```
unicode-data
Copyright 2020 Composewell Technologies and Contributors

This product includes software developed at
Composewell Technologies (http://www.composewell.com/).
```